core/src/services/compfs/backend.rs

```diff
@@ -63,6 +63,19 @@ impl Builder for CompfsBuilder {
                 "root is not specified",
             )),
         }?;
+
+        // If root dir is not exist, we must create it.
+        if let Err(e) = std::fs::metadata(&root) {
```

Comment:
Should we also dispatch this part? It's blocking.

Reply:
Builder is required to be blocking.